### PR TITLE
Kakscalculator2

### PR DESCRIPTION
**Pull request by kullrich**
Add kakscalculator2

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by kullrich** [comment]
@BiocondaBot please add label

File: recipes/kakscalculator2/build.sh
Comment (mbargull) [inline recipes/kakscalculator2/build.sh:4]:
```suggestion
make CC=${CXX} CFLAGS="${CXXFLAGS}"
```
To get the standard set of compiler flags we use.

File: recipes/kakscalculator2/build.sh
Comment (mbargull) [inline recipes/kakscalculator2/build.sh:4]:
```suggestion
make CC=${CXX} CFLAGS="${CXXFLAGS}"
```

**Review by mbargull — APPROVED** [review]
LGTM, thanks @kullrich!

**Comment by kullrich** [comment]
@mbargull Do I need to update my fork with the typo fix or will this be handled within this PR?

**Comment by mbargull** [comment]
My commit was directly to your fork's branch which this PR is associated with (by default GitHub has this "allow edits from maintainers" box checked).
You can just `git pull` locally if you want to have that change in your local copy.

**Comment by kullrich** [comment]
@mbargull Thanks, if the check goes fine I guess I do not need once it is merged into master.

**Comment by mbargull** [comment]
Correct, once it's in, it's in and you can retrieve everything from either repository :).

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

**Comment by mbargull** [comment]
I see there are two more binaries in there: `AXTConvertor` and `ConPairs`. Do you want to add tests for them too?

----
But in any case, this already looks good to me, so feel free to add a `@BiocondaBot, please merge` comment to have it merged!

**Comment by kullrich** [comment]
@BiocondaBot, please merge

**Pull request merged**
